### ISSUE TITLE
Dof of chi-squared field

### ISSUE THREAD

**Issue by andrewfowlie** [cranmer/look-elsewhere-2d#1]
I understand that the formula used for the expected Euler characteristic assumes that the random field is chi-squared with one degree of freedom (eq. 9 of Gross & Vitells).

The comments discuss searches for a new particle with a 2d LEE for the mass and width. In that case though, since the signal strength must be positive, shouldn’t we have a chi-squared distribution with only ‘half’ a degree of freedom? 

This isn’t a bug per se as the implementation of eq. 9 may be correct, but the documentation/comments could mention that this isn’t applicable to searches for new particles.

On top of that, I’m unsure about your interpretation of eq. 9. I think it might be poorly presented in Gross & Vitells. I suspect that first term in eq. 9 should correspond to a chi-squared with two dof, not one dof as in the code. The number of dof in that term is determined by the number of free parameters (mass and width).

In summary, for the particle physics case, we should have
`E[\phi] = 1/2 P(\chi^2_2 > u) + ...`
We have in the code
`E[\phi] = P(\chi^2_1 > u) + ...`
which may be incorrect.

**Comment by andrewfowlie** [comment]
Returning to this, see discussion around p66 of [this talk](https://indico.cern.ch/event/687488/attachments/1640343/2618894/LEE_CERN_lecture_3_2018.pdf) and summary slide 76.
